File: resources/samples.rb
#!/usr/bin/ruby


def sum(*n)
  n.reduce(:+)
end

def avg(*n)
  result = sum(*n) / n.length
  result
end

def plain(p)
  p
end

# {"module": "samples", "function": "avg", "arguments": [2,4,5,1,23]}

File: resources/samples.py
#!/usr/bin/python


def _sum(a, b):
    return int(a) + int(b)

def avg(*n):
    result = sum(map(int, n)) / len(n)
    return result

# {"module": "samples", "function": "avg", "arguments": [2,4,5,1,23]}
